InventorySales/Model/AreProductsSalable.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySales\Model;
+
+use Magento\InventorySalesApi\Api\AreProductsSalableInterface;
+use Magento\InventorySalesApi\Api\Data\AreProductsSalableResultInterface;
+use Magento\InventorySalesApi\Api\Data\AreProductsSalableResultInterfaceFactory;
+use Magento\InventorySalesApi\Api\Data\IsProductSalableResultInterfaceFactory;
+use Magento\InventorySalesApi\Api\IsProductSalableInterface;
+
+/**
+ * @inheritdoc
+ */
+class AreProductsSalable implements AreProductsSalableInterface
+{
+    /**
+     * @var IsProductSalableInterface
+     */
+    private $isProductSalable;
+
+    /**
+     * @var AreProductsSalableResultInterfaceFactory
+     */
+    private $areProductsSalableResultFactory;
+
+    /**
+     * @var IsProductSalableResultInterfaceFactory
+     */
+    private $isProductSalableResultFactory;
+
+    /**
+     * @param IsProductSalableInterface $isProductSalable
+     * @param IsProductSalableResultInterfaceFactory $isProductSalableResultFactory
+     * @param AreProductsSalableResultInterfaceFactory $areProductsSalableResultFactory
+     */
+    public function __construct(
+        IsProductSalableInterface $isProductSalable,
+        IsProductSalableResultInterfaceFactory $isProductSalableResultFactory,
+        AreProductsSalableResultInterfaceFactory $areProductsSalableResultFactory
+    ) {
+        $this->isProductSalable = $isProductSalable;
+        $this->isProductSalableResultFactory = $isProductSalableResultFactory;
+        $this->areProductsSalableResultFactory = $areProductsSalableResultFactory;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function execute(string $skus, int $stockId): AreProductsSalableResultInterface
```

Comment:
Why @skus is not an array?

Reply:
As far as I know, there is no ability to pass array via REST API for GET request.